android/src/main/kotlin/com/openlogin/flutter/openlogin_flutter/OpenloginFlutterPlugin.kt

```diff
@@ -139,30 +145,52 @@ class OpenloginFlutterPlugin: FlutterPlugin, ActivityAware, MethodCallHandler, P
     throw NotImplementedError()
   }
 
-  private fun getOpenLoginProvider(provider: String): OpenLogin.Provider {
+  private fun getOpenLoginProvider(provider: String): Provider {
     return when (provider) {
-      "google" -> OpenLogin.Provider.GOOGLE
-      "facebook" -> OpenLogin.Provider.FACEBOOK
-      "reddit" -> OpenLogin.Provider.REDDIT
-      "discord" -> OpenLogin.Provider.DISCORD
-      "twitch" -> OpenLogin.Provider.TWITCH
-      "apple" -> OpenLogin.Provider.APPLE
-      "line" -> OpenLogin.Provider.LINE
-      "github" -> OpenLogin.Provider.GITHUB
-      "kakao" -> OpenLogin.Provider.KAKAO
-      "linkedin" -> OpenLogin.Provider.LINKEDIN
-      "twitter" -> OpenLogin.Provider.TWITTER
-      "weibo" -> OpenLogin.Provider.WEIBO
-      "wechat" -> OpenLogin.Provider.WECHAT
-      "email_passwordless" -> OpenLogin.Provider.EMAIL_PASSWORDLESS
-
-      else -> OpenLogin.Provider.GOOGLE
+      "google" -> Provider.GOOGLE
+      "facebook" -> Provider.FACEBOOK
+      "reddit" -> Provider.REDDIT
+      "discord" -> Provider.DISCORD
+      "twitch" -> Provider.TWITCH
+      "apple" -> Provider.APPLE
+      "line" -> Provider.LINE
+      "github" -> Provider.GITHUB
+      "kakao" -> Provider.KAKAO
+      "linkedin" -> Provider.LINKEDIN
+      "twitter" -> Provider.TWITTER
+      "weibo" -> Provider.WEIBO
+      "wechat" -> Provider.WECHAT
+      "email_passwordless" -> Provider.EMAIL_PASSWORDLESS
+
+      else -> Provider.GOOGLE
     }
   }
 
-  private fun getOpenLoginNetwork(network : String) : OpenLogin.Network {
-    if (network.equals("mainnet", true)) return OpenLogin.Network.MAINNET
-    else return OpenLogin.Network.TESTNET
+  private fun getOpenLoginNetwork(network : String) : Web3Auth.Network {
+    if (network.equals("mainnet", true)) return Web3Auth.Network.MAINNET
```

Comment:
pls add cyan network support too
default is mainnet
so, if no network is specified, it should be mainnet

Reply:
done